tools/BUILD.bazel

```
@@ -13,6 +14,21 @@ ts_config(
     deps = ["tsconfig.json"],
 )
 
+rules_js_tsconfig(
+    name = "tsconfig",
```

Can we use the naming convention we used in components

```suggestion
    name = "build-tsconfig",
```